plugin/src/main/scala/com/softwaremill/clippy/AdviceLoader.scala

```diff
@@ -15,7 +15,8 @@ class AdviceLoader(global: Global, url: String, localStoreDir: File)(implicit ec
   private val OneDayMillis = 1000L * 60 * 60 * 24
 
   private val localStore = new File(localStoreDir, "clippy.json.gz")
-  private val projectAdviceFile = new File(".clippy.json")
+  private val projectRoot = sys.props.get("user.dir").getOrElse(".")
```

Comment:
I may have gotten this wrong, I thought user.dir was the same as the project dir.  That is what I am really after.  Alternatively, maybe I should just move it to resources and get rid of the mess of trying to figure out project dir at all?  I was thinking last night if they used guids instead of Ints for strings it would be easy enough to scan all *.clippy.json in the resources and merge them.  Would you be interested me me making this change?